desktop-src/dataxchg/html-clipboard-format.md

```diff
@@ -21,244 +21,346 @@ ms.date: 05/31/2018
 
-            &lt;/BODY&gt;&lt;/HTML&gt;The selection, as delimited by StartSelection and EndSelection, is shown in bold.
+> <TABLE BORDER>
+> 	<TR>
+> 		<TH ROWSPAN=2>Head 1</TH>
+> 		<TD>Item 1</TD>
+> 		<TD>Item 2</TD>
+> 		<TD>Item 3</TD>
+> 		<TD>Item 4</TD>
+> 	</TR>
+> 	<TR>
+> 		<TD>Item 5</TD>
+> 		<TD>Item 6</TD>
+> 		<TD>Item 7</TD>
+> 		<TD>Item 8</TD>
+> 	</TR>
+> 	<TR>
+> 		<TH>Head 2</TH>
+> 		<TD>Item 9</TD>
+> 		<TD>Item 10</TD>
+> 		<TD>Item 11</TD>
+> 		<TD>Item 12</TD>
+> 	</TR>
+> </TABLE>
 
-3.  Pasting a fragment of an ordered list into plain text.
-    -   -   HTML text:
 
-            &lt;BODY&gt;<OL TYPE = a><LI>Item 1<LI>Item 2<LI>Item 3<LI>Item 4<LI>Item 5<LI>Item 6</OL>&lt;/BODY&gt;
+#### How MSHTML handles copying a substring of HTML
 
-        -   Appears as:
-            1.  Item 1
-            2.  Item 2
-            3.  Item 3
-            4.  Item 4
-            5.  Item 5
-            6.  Item 6
-        -   The user selects and copies items 3 through 5 to the clipboard. The following HTML is in the clipboard:
+When the user uses their mouse to make a text selection covering the table cells <b>Item 6</b>, <b>Item 7</b>, <b>Item 10</b>, and <b>Item 11</b>. This selection is then copied into the clipboard.
 
-            <DOCTYPE...>&lt;HTML&gt;&lt;BODY&gt;<OL TYPE = a>
+What follows is what will be on the clipboard (note this is IE4/MSHTML's interpretation). Line-breaks have been added for clarity.
 
-            <!-- StartFragment-->
+```
+<!DOCTYPE
+<HTML>
+<BODY>
+<TABLE BORDER>
+<!--StartFragment-->
+	**<TR>
+		<TD>Item 6</TD>
+		<TD>Item 7</TD>
+	</TR>
+	<TR>
+		<TD>Item 10</TD>
+		<TD>Item 11</TD>
+	</TR>**
+<!--EndFragment-->
+</TABLE>
+</BODY>
+</HTML>
+```
 
-            **<LI>Item 3<LI>Item 4<LI>Item 5**
+The selection, as delimited by `StartSelection` and `EndSelection`, is shown in bold.
 
-            <!-- EndFragment-->
+### Scenario 3 - Pasting a fragment of an ordered list `<ol>` into plain text
 
-            </OL>&lt;/BODY&gt;&lt;/HTML&gt;
+<!-- TODO: If a user copies a middle subsection of an ordered list, and then pastes it elsewhere, should the values of those bullet-points be preserved? If so, then how should the DOM represent it? -->
 
-            The selection, as delimited by StartSelection and EndSelection, is show in bold.
+Assume the following HTML text:
 
-        -   If this fragment is now pasted into an empty document, the following HTML will be created:
+> ```
+> <BODY><OL TYPE="a"><LI>Item 1<LI>Item 2<LI>Item 3<LI>Item 4<LI>Item 5<LI>Item 6</OL></BODY>
+> ```
 
-            &lt;BODY&gt;<OL TYPE = a><LI>Item 3<LI>Item 4<LI>Item 5</OL>&lt;/BODY&gt;
+This will appear as follows:
 
-        -   Appearing as:
-            1.  Item 3
-            2.  Item 4
-            3.  Item 5
+<!-- or just let Markdown do its own thang?
+> <ol>
+> 	<li>Item 1</li>
+> 	<li>Item 2</li>
+> 	<li>Item 3</li>
+> 	<li>Item 4</li>
+> 	<li>Item 5</li>
+> 	<li>Item 6</li>
+> </ol>
+ -->
```

Comment:
```suggestion
```

The result in line 244 looks good in Markdown, see https://github.com/daiplusplus/win32/blob/patch-1/desktop-src/dataxchg/html-clipboard-format.md